### DISCUSSION TITLE
Option of a second window with folders and files

### DISCUSSION THREAD

**Discussion by snden** [vkbo/novelWriter#1136] [Ideas]
I'm currently working on a text that already has so many chapters that I have to scroll to get to the Characters or Objects folder. Then I scroll back up to the text file again. Would it be possible to create an option to have a second window with folders and files that would be identical in function? For example, under the existing window on the left. Option to turn this window on or off, depending on whether the user needs it or not. That way I could work in two places at the same time. If it were then possible to move files and folders between these two windows, that would be great.

**Comment by vkbo** [comment]
When a project is open, the entire structure of the project is kept in the structure of the project tree, so the way the code is written, splitting it up would be difficult and require a major rewrite. Due to the amount of things in the backlog, it is very unlikely I will have the time to do this in the foreseeable future.

I discovered very early on that the tree could potentially get very large, and require a lot of scrolling. That's why you can freely add folders into it to split it up in collapsible chunks. Personally, I split my text up into acts, which makes sense to my workflow.

Also, when you have defined your documents for a story, or a part of it, you can also access the content from the novel view, which would let you keep the Novel root folder collapsed in the Project Tree.

The next release will have numerous improvements to these two views, so it will be easier as well.

**Reply by vkbo** [reply]
By the way, I am considering adding a quick navigation view under the project tree that can be used to quickly navigate between headers of large note files. I could potentially also add a quick nav feature for the project tree itself. It could for instance let you quickly move between root folders.

**Reply by vkbo** [reply]
I had an idea, so added #1137. It is a simple feature to add, and would hopefully help with large projects.

**Reply by snden** [reply]
You're right that at this point it would help me to have the Characters and Objects folder listed in the project window and the folder I'm currently working in visible in the Novel window. I look forward to the changes in the new version.

**Reply by vkbo** [reply]
A major change in 1.7 is that you can add as many Novel folders as you want, and you will then be able to select which ones to show in the Novel View and Outline View. You will also have the option to show them all. The feature is intended for writing for instance trilogies, but there is no restriction on how you use them.
